testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/hello-world.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { credentials } from 'grpc';
+import { CustomerServiceClient, CustomerRequestId, Empty } from './src/grpc/helloworld';
+import chalk from "chalk";
+describe('hello world client checks via grpc', () => {
+    let client: CustomerServiceClient;
+    it('should check that jest is running', (done) => {
+        expect(1).toEqual(1);
```

Comment:
Do we really need to check 1=1? If Jest is working the rest of the test should be run.